TROUBLESHOOTING.md

```diff
@@ -36,7 +36,45 @@ to index for features such as go to definition.
 
 ## Common issues
 
-There are two main sources of issues users typically face during activation: shell or Bundler related problems.
+There are several main sources of issues users typically face during activation: outdated version, shell problems, or Bundler related problems.
+
+### Outdated Version
+
+> [!NOTE] If using VS Code, the version of the extension is distinct from that that of the server (the `ruby-lsp` gem). You can check the server version in the Output panel upon startup.
```

Comment:
```suggestion
> [!NOTE] If using VS Code, the version of the extension is distinct from that of the server (the `ruby-lsp` gem). You can check the server version in the Output panel upon startup.
```

Reply:
Should we also include a screenshot of control panel?